applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/app_ZReleaseMCUBoot.conf

```diff
@@ -129,3 +129,4 @@ CONFIG_WATCHDOG=y
 # Bootloader Configuration
 
 CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_BOOT_SIGNATURE_KEY_FILE="root-rsa-2048.pem"
```

Comment:
if we are adding an explicit key, maybe we should use the same key as for b0?

Reply:
B0 actually uses Elliptic Curve Key, so we cannot use it.
The alternative solution could be to generate our own RSA key with given length, but we always used the default key for MCUBoot.

Reply:
Where it will look for the key? If it is the default mcuboot directory then this change gives little improvement...

Reply:
Yes, it will look for the key in the default MCUBoot directory.

The change is done to make user aware that he should change the key file path also in the bootloader configuration if he uses another key file (its also more consistent to define key file path here if we define it in the application configuration).

Reply:
Looking for key in MCUBoot directory is currently hardcoded, changing it would require some changes in build system.